### DISCUSSION TITLE
[express-serve-static-core] has no exported member Locals

### DISCUSSION THREAD

**Discussion by beerpacks** [DefinitelyTyped/DefinitelyTyped#64167] [Issues with a @types package]
Since the last update of @types/express, I have the following error : node_modules/@types/express/index.d.ts:112:35 - error TS2694: Namespace '"{project path}/node_modules/@types/express-serve-static-core/index"' has no exported member 'Locals'. 

I have look the pull request and it has been deleted from the file @types/express-serve-static-core.

[a comment by an automated account is not shown]

**Reply by TabotCharlesBessong** [reply]
Module '"express-serve-static-core"' has no exported member 'Query'

**Comment by m1cm1c** [comment]
I'm experiencing the same problem. It occurs with @types/express@4.17.16 but does not occur with @types/express@4.17.15.

**Reply by carl-parrish** [reply]
Thanks this helped me a lot but hopefully a this gets fixed in a future release.
